ML-TOOLS.md

```diff
@@ -0,0 +1,88 @@
+| **[LLaMA](https://ai.meta.com/llama/)**                  | Meta's open-weight large language model family. Popular for research and fine-tuning.         | `open`, `research`, `local`, `LLM`         |
+| **[GPT-Neo/GPT-J](https://huggingface.co/EleutherAI)**   | Open LLMs from EleutherAI. Can be run locally or hosted via HuggingFace.                      | `LLM`, `open`, `education`, `poetry`       |
+| **[StableLM](https://github.com/Stability-AI/StableLM)** | Stability AI's open-source LLM series for local generation tasks.                             | `creative`, `open`, `local`, `LLM`         |
+
+---
+
+## 🌐 API-Only / Cloud-Based Models
+
+Require API keys or subscriptions to use—less transparent, but powerful and production-ready.
+
+| Tool/Platform                                                     | Description                                                              | Tags                                             |
+| ----------------------------------------------------------------- | ------------------------------------------------------------------------ | ------------------------------------------------ |
+| **[OpenAI GPT-4 / ChatGPT](https://platform.openai.com/)**        | Leading closed-source LLM with natural dialogue and multi-modal support. | `closed`, `LLM`, `creative`, `education`         |
+| **[Google Gemini](https://deepmind.google/technologies/gemini/)** | Multimodal models for voice, vision, and language (via Bard or API).     | `closed`, `TTS/voice`, `AI writing`, `education` |
+| **[RunwayML](https://runwayml.com/)**                             | GUI-based creative ML platform—GPT, TTS, image gen, video tools.         | `closed`, `GUI`, `TTS`, `visual`, `creative`     |
+| **[Cohere](https://cohere.com/)**                                 | API-based NLP tools for classification, generation, and embeddings.      | `LLM`, `commercial`, `API`, `education`          |
+
+---
+
+## 🎨 Local & Artist-Friendly Tools
+
+These are especially useful for workshops, teaching, or creative coding environments.
+
+| Tool                                                              | Description                                                                     | Tags                                               |
+| ----------------------------------------------------------------- | ------------------------------------------------------------------------------- | -------------------------------------------------- |
+| **[ml5.js](https://ml5js.org/)**                                  | Beginner-friendly ML for the web using JavaScript and p5.js. Great for artists. | `creative`, `education`, `TTS`, `vision`, `open`   |
+| **[Wekinator](http://www.wekinator.org/)**                        | GUI for training ML models with real-time input/output (e.g., sensors).         | `creative`, `education`, `interaction`, `gesture`  |
```

Comment:
I don't know if wekinator is actively maintained and can still easily be run / integrated with creative coding tools. But I do love to refer to it!